modules/channel-web/src/views/lite/main.jsx

```diff
@@ -600,8 +600,11 @@ export default class Web extends React.Component {
       return null
     }
 
-    window.parent &&
-      window.parent.postMessage({ type: 'setClass', value: 'bp-widget-web bp-widget-' + this.state.view }, '*')
+    const parentClass = `bp-widget-web bp-widget-${this.state.view}`
+    if (this.state.parentClass !== parentClass) {
+      window.parent && window.parent.postMessage({ type: 'setClass', value: parentClass }, '*')
+      this.setState({ parentClass })
```

Comment:
why set in state?  seems like you don't need it for rendering

Reply:
Fixed that